packages/editor/src/components/post-status/style.scss

```diff
@@ -4,6 +4,11 @@
 	&.is-read-only {
 		padding: 6px 12px;
 	}
+
+	.editor-post-status__toggle.editor-post-status__toggle {
+		padding-top: $grid-unit-05;
+		padding-bottom: $grid-unit-05;
+	}
```

Comment:
In order to override the padding on the button component, we need to increase the CSS specificity to `0-3-0`.

![image](https://github.com/user-attachments/assets/9a70baf7-4cdf-46af-a38c-29872ac3c85c)